### ISSUE TITLE
Master branch missing commit history

### ISSUE THREAD

**Issue by sparrowt** [microsoft/mssql-django#129]
As noticed and raised here https://github.com/microsoft/mssql-django/issues/86#issuecomment-1116247696 I think the master branch has been - perhaps unintentially - deprived of some useful commit history due to #124 being squash merged.

Resulting issues:
 - git blame will be less useful in future on master with changes being attributed to the "Prepare for 1.1.3 release" commit instead of the original commit which references the issue & explains what was being fixed
 - commits appear to be missing from master even though in fact they are on there, e.g. https://github.com/microsoft/mssql-django/compare/master...dev is far longer than it should be
 - also the reverse diff is non-zero which seems wrong https://github.com/microsoft/mssql-django/compare/dev...master - dev should have everything that's on master (+ potentially new unrelease stuff) so there shouldn't be code changes on master which show up as being missing from dev

One set of steps to resolve this would be:
1. force push master to HEAD~1 (i.e. 'undo' c0476cf4e49ab3dcbbab37ccb3e558216841b6dc)
2. then create a new branch checked out from 1779a242593539241a43a1eaf24e5ed17ba586c3 (where `dev` was when #124 was opened)
3. create a PR from this new branch to master and merge it in a non-squashing way

Any thoughts?

**Comment by jmah8** [comment]
Yes, you are correct about that. We accidentally squash committed. We are planning to fix it during the next release, but we can fix it sooner if you prefer.

**Comment by jmah8** [comment]
This is fixed by PR #146. I am going to close this issue, but feel free to reopen it if there are still concerns about the commit history.
